.npmignore

```diff
@@ -1 +1,32 @@
 node-tests/
+CODE_OF_CONDUCT.md
+CONTRIBUTING.md
+FEATURES.md
+STYLEGUIDE.md
+VERSION
+bin
+blueprints
+bower.json
+bower_components
+config
+# dist
```

Comment:
It's just a comment. In the future we might want to only ship certain things inside `/dist`